src/AutoRest.CSharp/Common/Output/Models/Types/SchemaObjectType.cs

```diff
@@ -554,7 +554,7 @@ public ObjectTypeProperty GetPropertyForGroupedParameter(RequestParameter groupe
             return objectProperty;
         }
 
-        private bool TryGetPropertyForSchemaProperty(Func<ObjectTypeProperty, bool> propertySelector, [NotNullWhen(true)] out ObjectTypeProperty? objectProperty, bool includeParents = false)
+        protected bool TryGetPropertyForSchemaProperty(Func<ObjectTypeProperty, bool> propertySelector, [NotNullWhen(true)] out ObjectTypeProperty? objectProperty, bool includeParents = false)
```

Comment:
Is this used in the overriding type?

Reply:
Yes, in `MgmtObjectType.GetPropertyForSchemaProperty` below.

Reply:
I don't see `TryGetPropertyForSchemaProperty` used there.

Reply:
I think the Get* methods are called but not the TryGet, looks like we can revert this line

Reply:
You are correct. I forgot to roll back the changes here. Previously I copied the codes and then refactored to remove them, but forgot to change the calling method visibility.